RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitSet.java

```diff
@@ -71,9 +72,15 @@ public boolean get(int bitIndex) {
 
   @Override
   public BitSet get(int fromIndex, int toIndex) {
-    BitSet bitSet = BitSetUtil.bitsetOf(roaringBitmap);
-    bitSet = bitSet.get(fromIndex, toIndex);
-    return new RoaringBitSet(fromBitSet(bitSet));
+    return new RoaringBitSet(
```

Comment:
Why not use `BitSetUtil.bitsetOf(RoaringBitmap.bitmapOfRange(fromIndex, toIndex))`?

Note that it would be safer to validate the indexes first. E.g., at a minimum they should be positive integers.

Reply:
I think `RoaringBitmap.or` is entirely unnecessary here.

Reply:
Thanks for the quick review, @lemire.

> Why not use `BitSetUtil.bitsetOf(RoaringBitmap.bitmapOfRange(fromIndex, toIndex))`?

Does that provide a way to achieve the same result? `RoaringBitmap.bitmapOfRange` is a static method. Is there a way other to get a range of a RoaringBitmap instance? That's why I used `RoaringBitmap.or` since I didn't find another way.

> Note that it would be safer to validate the indexes first. E.g., at a minimum they should be positive integers.

Thanks, I added a check.

Reply:
> Does that provide a way to achieve the same result? RoaringBitmap.bitmapOfRange is a static method.  Is there a way to get a range of a RoaringBitmap instance? That's why I used RoaringBitmap.or(Iterator<? extends RoaringBitmap> bitmaps, long rangeStart, long rangeEnd) since I didn't find another way.

RoaringBitmap.bitmapOfRange and RoaringBitmap.or are both static methods.

"I didn't find another way" ... the other way is RoaringBitmap.bitmapOfRange. :-)

No matter how you write this, you will first create a new Roaring Bitmap with just the range of values you need, followed by the create of a new (and final) bitmap that is offsetted to zero. So I don't think you can make it more efficient.

But if you use RoaringBitmap.bitmapOfRange, your code is going to be simpler... which is what I suggest you go for.

Reply:
@lemire Thanks for explaining. I pushed a new commit. PTAL